zerver/lib/actions.py

```diff
@@ -1248,7 +1256,7 @@ def send_pm_if_empty_stream(sender, stream, stream_name, realm):
                "click the gear in the left-side stream list." %
                (sender.full_name, stream_name, error_msg))
     message = internal_prep_message(settings.NOTIFICATION_BOT, "private",
-                                    sender.bot_owner.email, "", content)
+                                    sender.bot_owner.email, "", content, realm)
```

Comment:
please spell this as `, realm=realm)` -- that's a more robust way to call an optional keyword argument.